File: VB/CustomLookUpEdit.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.Xpf.Grid.LookUp
Imports DevExpress.Xpf.Editors.Settings

Namespace AccentInsensitiveSearch
	Public Class CustomLookUpEdit
		Inherits LookUpEdit
		Protected Overrides Function CreateEditorSettings() As BaseEditSettings
			Return New AccentInsensitiveEditSettings()
		End Function
	End Class
End Namespace

File: CS/CustomLookUpEdit.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.Xpf.Grid.LookUp;
using DevExpress.Xpf.Editors.Settings;

namespace AccentInsensitiveSearch {
    public class CustomLookUpEdit : LookUpEdit {
        protected override BaseEditSettings CreateEditorSettings() {
            return new AccentInsensitiveEditSettings();
        }
    }
}
